src/Pages/category/Component/CategoryList/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+import CategoryListItem from './CategoryListItem';
+
+function CategoryList({
+  categoryList,
+  onChangeClickId,
+  handleOnchangeMenuFocus,
+  chgColorMenu,
+}) {
+  return (
+    <ListContainer>
+      <CategoryListWrap>
+        <h1>가구</h1>
+        <CategoryListItem
+          categoryList={categoryList}
+          onChangeClickId={onChangeClickId}
+          handleOnchangeMenuFocus={handleOnchangeMenuFocus}
+          chgColorMenu={chgColorMenu}
+        />
+      </CategoryListWrap>
+    </ListContainer>
```

Comment:
```suggestion
function CategoryList(props) {
  return (
    <ListContainer>
      <CategoryListWrap>
        <h1>가구</h1>
        <CategoryListItem {...props} />
      </CategoryListWrap>
    </ListContainer>
```
언제나 권장되는 패턴은 아니지만 이렇게도 사용 가능합니다

Reply:
넵 혹시 언제나 권장되는 패턴이 아니라는 것은 전체 props를 보내는 것이 아닌 필요한 props만 보낼 때를 말씀하는 것이 맞나요??

위 방식으로 하면 모든 props를 내려주기 때문에 필요하지 않은 props도 같이 내려가서 지금은 모든 props를 쓰기 때문에 가능한 것이 맞죠??

Reply:
@SON-JUYOUNG 구웃~